### ISSUE TITLE
Use the import API to keep timestamps history from Freshdesk

### ISSUE THREAD

**Issue by skipjac** [mittsh/freshdesk-zendesk-migration#1]
There is a import API that will allow you to keep all the timestamps and import a ticket with all the comments in one call. 

http://developer.zendesk.com/documentation/rest_api/ticket_import.html

**Comment by mittsh** [comment]
Wow! I should have notice this one before... Thanks!
Though, I've already imported all my tickets, so I don't think I'll spend any more time on this code. If anyone feels inspired, feel free to fork, make a PR (or not!). I won't be supporting this code so much anyways. If you want, I can transfer you or @Zendesk the ownership.

**Comment by blak3r** [comment]
I modified the script to do this plus some other stuff...
Sorry not enough time to do a pull request.  Here's a gist for others:

https://gist.github.com/blak3r/093fc9ad71e99855c899

**Comment by mittsh** [comment]
Thanks for sharing @blak3r! I'm frankly not supporting this repo anymore. But I guess it will help others...

**Comment by blak3r** [comment]
Yep yep. Totally understand. Thanks for the great starting point.
